docs/source/guides/using_koalas_entitysets.rst

```diff
@@ -0,0 +1,115 @@
+Using Koalas EntitySets (BETA)
+==============================
+.. note::
+    Support for Koalas EntitySets is still in Beta. While the key functionality has been implemented, development is ongoing to add the remaining functionality.
+
+    All planned improvements to the Featuretools/Koalas integration are `documented on Github <https://github.com/FeatureLabs/featuretools/issues?q=is%3Aopen+is%3Aissue+label%3AKoalas>`_. If you see an open issue that is important for your application, please let us know by upvoting or commenting on the issue. If you encounter any errors using Koalas entities, or find missing functionality that does not yet have an open issue, please create a `new issue on Github <https://github.com/FeatureLabs/featuretools/issues>`_.
+
+Creating a feature matrix from a very large dataset can be problematic if the underlying pandas dataframes that make up the entities cannot easily fit in memory. To help get around this issue, Featuretools supports creating ``Entity`` and ``EntitySet`` objects from Koalas dataframes. A Koalas ``EntitySet`` can then be passed to ``featuretools.dfs`` or ``featuretools.calculate_feature_matrix`` to create a feature matrix, which will be returned as a Koalas dataframe. In addition to working on larger than memory datasets, this approach also allows users to take advantage of the parallel and distributed processing capabilities offered by Koalas and Spark.
+
+This guide will provide an overview of how to create a Koalas ``EntitySet`` and then generate a feature matrix from it. If you are already familiar with creating a feature matrix starting from pandas dataframes, this process will seem quite familiar, as there are no differences in the process. There are, however, some limitations when using Koalas dataframes, and those limitations are reviewed in more detail below.
+
+Creating Entities and EntitySets
+--------------------------------
+Koalas ``EntitySets`` require Koalas and PySpark. Both can be installed directly with ``pip install featuretools[koalas]``. Java is also required for PySpark and may need to be installed, see `the Spark documentation <https://spark.apache.org/docs/latest/index.html>`_ for more details. We will create a very small Koalas dataframe for this example. Koalas dataframes can also be created from pandas dataframes, or read in directly from a file.
```

Comment:
It's obvious, but I'd mention that Koalas dataframe can also be created from Spark dataframes.